template.yaml

```diff
@@ -45,14 +45,17 @@ Parameters:
   MarketplaceTechAdminEmail:
     Type: String
     AllowedPattern: ".*"
+  
+  MarketplaceSellerEmail:
```

Comment:
This parameter should be optional. What if I want to use my own service for sending emails?